app/customer/customer_class.py

```diff
@@ -0,0 +1,85 @@
+from app.car.car_class import Car
+
+
+class Customer:
+    def __init__(self, customer: dict, fuel_price: str) -> None:
+        self.name = customer["name"]
+        self.cart = customer["product_cart"]
+        self.location = customer["location"]
+        self.money = customer["money"]
+        self.car = Car(customer["car"], float(fuel_price))
+
+    def go_to_shop(self, shops: dict) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+        shop = self.cost_of_trip(shops)
+
+        if shop:
+            self.cost_of_product(shop, True)
+
+    def cost_of_way(self, shop_location: list) -> float:
+        return self.car.cost_of_way(self.location, shop_location)
+
+    def cost_of_product(
+            self,
+            shop: dict,
+            detail: bool = False
+    ) -> list[float, dict]:
+        if detail:
+            print(
+                f"Date: 04/01/2021 12:33:41\n"
+                f"Thanks, {self.name}, for your purchase!\n"
+                f"You have bought: "
+            )
+
+        sum_of_prices = 0
+
+        for key, value in shop["products"].items():
```

Comment:
add more meaningful name

Reply:
The following comment is not resolved in this PR.